xonsh/history.py

```diff
@@ -432,26 +591,30 @@ def _gc(ns, hist):
 
 
 _MAIN_ACTIONS = {
-    'show': _show,
+    'session': History.show,
+    'all': History.show,
+    'zsh': History.show,    
+    'bash': History.show,    
     'id': lambda ns, hist: print(hist.sessionid),
     'file': lambda ns, hist: print(hist.filename),
     'info': _info,
     'diff': diff_history._main_action,
-    'gc': _gc,
+    'gc': _gc
```

Comment:
Generally it makes editing easier -- if you rearrange a list you don't accidentally introduce syntax errors when you transpose lines.  Also it makes for cleaner diffs when you add items to a list.